src/app/components/SettingsTab.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * TCD Software
+ * Created by Dmitrij Rysanow on 02.03.17.
+ */
+import React from 'react';
+
+class SettingsTab extends React.Component {
+    render() {
+        return (
+            <p>testSettingsTab</p>
```

Comment:
I know that piece of component is totally temporary but add some padding or margin - text should not "stick" to the border - you can find it in documentation of material design (NOTE: not material-ui docs :) )